ui/components/app/alert-system/confirm-alert-modal/confirm-alert-modal.tsx

```diff
@@ -170,14 +163,22 @@ export function ConfirmAlertModal({
       onClose={onClose}
       customTitle={t('confirmAlertModalTitle')}
       customDetails={
-        <ConfirmDetails onAlertLinkClick={handleOpenMultipleAlertModal} />
+        selectedAlert?.provider === SecurityProvider.Blockaid ? (
+          SecurityProvider.Blockaid
+        ) : (
+          <ConfirmDetails onAlertLinkClick={handleOpenMultipleAlertModal} />
+        )
       }
       customAcknowledgeCheckbox={
         <AcknowledgeCheckboxBase
           selectedAlert={selectedAlert}
           isConfirmed={confirmCheckbox}
           onCheckboxClick={handleConfirmCheckbox}
-          label={t('confirmAlertModalAcknowledge')}
+          label={
+            SecurityProvider.Blockaid
```

Comment:
As an aside, should this ternary be comparing against the `selectedAlert.provider`?

This appears to be checking if an enum property exists which will always be `true`?

Reply:
Fixed 👍